src/apps/datacopy.c

```diff
@@ -411,7 +411,7 @@ create_target_table(char *sobjname, char *owner, char *dobjname, DBPROCESS * dbs
 	DBINT num_cols;
 	DBCOL2 colinfo;
 
-	sprintf(ls_command, "SET FMTONLY ON select * from %s SET FMTONLY OFF", sobjname);
+	sprintf(ls_command, "select top 0 * from %s", sobjname);
```

Comment:
why not using the more classic `select * from <table> where 0=1` ?

Reply:
Because it works fine (on Sybase and MSSQL). What you propose is ANSI compliant (was this your point?), but I'm not sure that's something very important here (we're on Free**TDS** after all).

Reply:
More than ANSI I think very old MSSQL/ASE databases does not support `TOP` (not sure about ASA either). But as far as I know they should support `SET FMTONLY`.

Reply:
Make sens. It's a free and safe change anyways, so I will do it too. And yes, I remember `select top` was introduced in Sybase 15 (I think?).

Reply:
Fixed.